src/routes/__tests__/home.test.js

```diff
@@ -0,0 +1,32 @@
+const home = require("../home");
+
+describe("Test data", () => {
+  const dataOriginal = {
```

Comment:
Let's try to keep a proper English, so **originalData** would be nicer. Also on the line 17.